cmd/dvotenode/dvotenode.go

```diff
@@ -117,6 +117,7 @@ func newConfig() (*config.DvoteCfg, config.Error) {
 	globalCfg.VochainConfig.MempoolSize = *flag.Int("vochainMempoolSize", 20000, "vochain mempool size")
 	globalCfg.VochainConfig.KeyKeeperIndex = *flag.Int8("keyKeeperIndex", 0, "if this node is a key keeper, use this index slot")
 	globalCfg.VochainConfig.ImportPreviousCensus = *flag.Bool("importPreviousCensus", false, "if enabled the census downloader will import all existing census")
+	globalCfg.VochainConfig.SecureTransactorList = *flag.StringArray("secureList", []string{}, "if empty any ethereum address can create and Oracle tx, else only the addresses on the list")
```

Comment:
I'd rather use this naming and description:
`ethereumWhiteListAddrs: list of allowed ethereum address for creating processes on the vochain (oracle mode only)`